gitlab/conf/runner_config.toml

```diff
@@ -0,0 +1,22 @@
+{% from "gitlab/map.jinja" import runner with context %}
```

Comment:
You should strip whitespaces with `{%-` and `-%}`

Reply:
standard is just` {%-`, i have bad exps w/ `-%}`

Reply:
You need to use `-%}` at first line otherwise you will have first line of file empty.